packages/coinstac-api-server/src/index.js

```diff
@@ -27,6 +27,7 @@ async function startServer() {
     },
     subscriptions: {
       async onConnect(connectionParams, ws) {
+        debugger
```

Comment:
?

Reply:
nah just from when I pushed for Ed, removed!